README.md

```
@@ -1 +1,27 @@
+히어릿은 개발자와 IT 종사자들이 빠르게 변화하는 IT 트렌드에 따라 원하는 정보를 더 이상 읽지 않고, **쉽게 들을 수 있는 팟캐스트 플랫폼**입니다.
+
+여기서 `히어릿(hEARit)`은 하나의 음성 콘텐츠를 뜻하며, 정보가 마치 스스로 들려주는 듯한 새로운 방식으로 사용자에게 다가갑니다.
+
+
+## Team
+### Backend
+
+| 사나(조은산) | 멍구(이유영) | 벡터(백승주) | 가콩(최가빈) |
+|--------|--------|--------|--------|
+| <div align="center">[JO-eusan](https://github.com/JO-eusan)</div> | <div align="center">[YuyoungRhee](https://github.com/YuyoungRhee)</div> | <div align="center">[Byesol](https://github.com/Byesol)</div> | <div align="center">[gabean13](https://github.com/gabean13)</div> |
+| <img src="https://avatars.githubusercontent.com/JO-eusan" width="150"/> | <img src="https://avatars.githubusercontent.com/YuyoungRhee" width="150"/> | <img src="https://avatars.githubusercontent.com/Byesol" width="150"/> | <img src="https://avatars.githubusercontent.com/gabean13" width="150"/> |
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)